feedback/broker_test.go

```diff
@@ -72,7 +72,8 @@ var _ = Describe("Broker", func() {
 
 		Describe("APNS Feedback Messages", func() {
 			Describe("Invalid Token", func() {
-				deviceToken := "CO8NP5B4PP51YVZ7FDMZI8QBLVI5HCFJRDK3YDOCLTFC9QOOJXVC2NNR8OM2UG5Y"
+				deviceToken :=
```

Comment:
If this intended? If so, consider adding a comment to disable the linting rule for the line, or break the large string with concatenation.